core/domain/feedback_services_test.py

```diff
@@ -400,6 +401,33 @@ def test_get_thread_summaries(self):
             # Check if the summaries match.
             self.assertEqual(thread_summary, thread_summaries[index])
 
+    def test_get_thread_summaries_load_test(self):
+        # The speed of fetching the summaries of 100 threads having 5 messages
+        # should be less than 8 seconds.
```

Comment:
40 threads having 5 messages comes under 1 second. 40 also seems like a good number. So should we fetch 40 threads?